src/sentry/static/sentry/app/components/assistant/guideAnchor.jsx

```diff
@@ -79,6 +94,12 @@ const GuideAnchor = createReactClass({
   },
 });
 
+export const conditionallyGuideAnchor = (condition, target, type, children) => {
```

Comment:
conditionally sounds weird. rename to `conditionGuideAnchor`?

Reply:
sure